packages/c-visually-hidden/tests/c-visually-hidden.cy.tsx

```diff
@@ -0,0 +1,22 @@
+import * as Examples from '../examples'
+import { h } from 'vue'
+
+describe('Visually Hidden Examples', () => {
+  it('hides text properly', () => {
+    cy.mount(Examples.BaseVisuallyHidden.default)
+      .get(document.body)
+      .contains('Visually Hidden')
+      .get('[data-testid=hidden]')
+      .should('have.css', {
+        border: '0px',
+        clip: 'rect(0px, 0px, 0px, 0px)',
+        height: '1px',
+        width: '1px',
+        margin: '-1px',
+        padding: '0px',
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        position: 'absolute',
+      })
```

Comment:
👍  I appreciate that Cypress has this API internally. Had to write one ourselves earlier on.

Reply:
Yeah I saw that. A better solution is for Cypress to handle this solution in its `visible` check. We already do a lot of logic in there to make sure z-index and transitions aren't affecting the actual visibility of the elements, so I'd argue that this is a usecase that should be solved natively in Cypress.